### DISCUSSION TITLE
Bootstrap / JavaScript issue on Github Pages

### DISCUSSION THREAD

**Discussion by Gabriel-Moraes-CAD** [community/community#29545] [Other Feature Feedback, Questions, & Ideas]
Hello,

I'm having issues with my Github page.
I created a project to post on my portfolio (hosted by Github pages) but I can't seem to get the JavaScript or Bootstrap 5 (CSS) to work.
I'm new to Web development and can't resolve the issue.
Thanks in advance for your help!

Repository link: https://github.com/Gabriel-Moraes-CAD/Gabriel-Moraes
file: basketball.html, basketball.js, basketball.css.
I also created a second version without boostrap 5 that also did not work properly. 
file:  basketball2.html, basketball.js, basketball2.css.

**Comment by mazkdevf** [comment]
> Hello,
> 
> I'm having issues with my Github page. I created a project to post on my portfolio (hosted by Github pages) but I can't seem to get the JavaScript or Bootstrap 5 (CSS) to work. I'm new to Web development and can't resolve the issue. Thanks in advance for your help!
> 
> Repository link: https://github.com/Gabriel-Moraes-CAD/Gabriel-Moraes file: basketball.html, basketball.js, basketball.css. I also created a second version without boostrap 5 that also did not work properly. file: basketball2.html, basketball.js, basketball2.css.

Hello!

**Reply by mazkdevf** [reply]
I will check the source out.

**Comment by mazkdevf** [comment]
I Found the issue, Creating Pull Request...

**Reply by Gabriel-Moraes-CAD** [reply]
That's awesome, thanks for your help. 
I look forward to learning more about the issue so I can prevent it in the future.

**Reply by mazkdevf** [reply]
https://github.com/Gabriel-Moraes-CAD/Gabriel-Moraes/pull/1

**Reply by mazkdevf** [reply]
> [Gabriel-Moraes-CAD/Gabriel-Moraes#1](https://github.com/Gabriel-Moraes-CAD/Gabriel-Moraes/pull/1)

Here is the Pull Request

**Reply by mazkdevf** [reply]
If that doesn't work just lemme know, I tried it with Private Repository GitHub Pages and it worked.

**Reply by Gabriel-Moraes-CAD** [reply]
Thanks a lot for your help! How did you find the issues?

**Reply by mazkdevf** [reply]
Well you had alot of these `/images` and to use it from it you should use the root directory fo that folder where is it located for example

if you have 

`index.html` and then you have folder named `assets`

you need to use `./assets`

if you have folder and it the assets folder is before that folder use `../assets`

**Reply by Gabriel-Moraes-CAD** [reply]
awesome, thanks again!

**Reply by mazkdevf** [reply]
> 

np, have a good day!
